exchange/utils.go

```diff
@@ -127,6 +133,36 @@ func cleanOpenRTBRequests(ctx context.Context,
 	return
 }
 
+// GDPRAccountEnabled returns the GDPR setting for the request type in the account config if defined or
+// the general GDPR setting in the account config if defined; otherwise it returns nil
+func accountLevelGDPREnabled(account *config.Account, requestType pbsmetrics.RequestType) *bool {
```

Comment:
Would this be better situated as a receiver of config.Account.GDPR?

```
account.GDPR.EnabledForRequestType(requestType pbsmetrics.RequestType) *bool
```

.. or with my refactor comment above, this could still be simplified to:

```
func accountLevelGDPREnabled(accountConfig *config.Account, privacyConfig config.Privacy, requestType pbsmetrics.RequestType) bool {
    if x := accountConfig.GDPR.EnabledForRequestType(requestType); x != nil {
        return *x
    }
    return privacyConfig.GDPR.Enabled
}
```

Reply:
Ah yep I was just working on CCPA thinking about possibly refactoring that so it's a receiver; I'll move forward with that change.